build-tools/enumification-helpers/README

```diff
@@ -0,0 +1,256 @@
+
+This directory contains a couple of "helper" tools for API int-to-enum conversion.
```

Comment:
This documentation is not really in quality. I'd rather "completely remove" this if it were to go there and supposed to "become for everyone".